http/media/multipart/src/main/java/io/helidon/http/media/multipart/ReadablePartLength.java

```diff
@@ -105,7 +105,7 @@ public int read() {
             }
 
             partRemaining--;
-            return dataReader.read();
+            return dataReader.read() & 0xFF;
```

Comment:
Please create an issue and fix in a separate PR.

Is it feasible to write a unit test ?